contracts/src/ethereum/TokenBridge.sol

```diff
@@ -0,0 +1,167 @@
+// SPDX-License-Identifier: Apache-2.0.
+pragma solidity ^0.6.12;
+
+import "@joriksch/sg-contracts/src/starkware/contracts/components/GenericGovernance.sol";
+import "@joriksch/sg-contracts/src/starkware/contracts/interfaces/ContractInitializer.sol";
+import "@joriksch/sg-contracts/src/starkware/contracts/interfaces/ProxySupport.sol";
+import "@joriksch/sg-contracts/src/starkware/cairo/eth/CairoConstants.sol";
+import "@joriksch/sg-contracts/src/starkware/starknet/eth/StarknetMessaging.sol";
+
+interface IL1Token {
```

Comment:
Can we just use IERC20 here?

Reply:
Definitely, I've created other functions while implementing but now IERC20 is indeed enough.